TechAndCheck/hypatia
# Bump rails to 7.0.2.2 and ruby to 3.1.1

This just bumps the two versions.

To test:
1. Install Ruby 3.1.1 locally (however you do it)
2. `bundle install`
3. `rails t` should just work.
